spec/parsers/bulkrax/bagit_parser_spec.rb

```diff
@@ -5,6 +5,34 @@
 
 module Bulkrax
   RSpec.describe BagitParser do
+    describe '#total' do
+      context 'while exporting' do
+        subject { described_class.new(exporter) }
+        let(:exporter) { FactoryBot.create(:bulkrax_exporter_worktype_bagit) }
+        let(:bulkrax_exporter_run) { FactoryBot.create(:bulkrax_exporter_run, exporter: exporter) }
+        let(:work_ids_solr) { [OpenStruct.new(id: SecureRandom.alphanumeric(9)), OpenStruct.new(id: SecureRandom.alphanumeric(9))] }
+        let(:collection_ids_solr) { [OpenStruct.new(id: SecureRandom.alphanumeric(9))] }
+        let(:file_set_ids_solr) { [OpenStruct.new(id: SecureRandom.alphanumeric(9)), OpenStruct.new(id: SecureRandom.alphanumeric(9)), OpenStruct.new(id: SecureRandom.alphanumeric(9))] }
+
+        before do
+          allow(subject).to receive(:current_record_ids).and_return(work_ids_solr + collection_ids_solr + file_set_ids_solr)
```

Comment:
we don't export collections with the bagit parser. so the totals below would all need to be minus 1.

Reply:
Oh good catch

Reply:
and the "collection_ids_solr" wouldn't need to be returned in this before block.